src/main/java/io/strimzi/kafka/bridge/SourceBridgeEndpoint.java

```diff
@@ -102,6 +103,9 @@ public void open() {
         Properties props = new Properties();
         props.putAll(kafkaConfig.getConfig());
         props.putAll(kafkaConfig.getProducerConfig().getConfig());
+        if (this.bridgeConfig.getTracing() != null && "jaeger".equals(this.bridgeConfig.getTracing())) {
```

Comment:
I suspect that this will not be dependent on the tyke of the tracing client. Every client supporting OpenTracing will use this. So maybe `!= null` should be enough?

Reply:
Yes you are right.